### PR TITLE
LowPtElectrons: NanoAOD integration (forward port)

### PR DESCRIPTION
**Pull request by bainbrid**
#### PR description:

This PR adds some minor fixes to the python config that steers the addition of low-pT electrons to the NanoAOD content, done in #33817.

The same minor fixes are already included in the back port PR #33992 to 10_6_X.

This PR ensures consistency between 10_6_X and master.

The changes are as follows:

- the `LeptonUpdater` module was mistakenly removed from `PhysicsTools/NanoAOD/python/lowPtElectrons_cff.py` in an earlier simplification to #33817; here it is added back with the param `computeMiniIso=True`, required to correctly compute the variables `miniPFRelIso_chg` and `miniPFRelIso_all`.
- the low-pT nano DQM plots have been updated: `miniPFRelIso_chg` and `miniPFRelIso_all` have a minor axis range tweak; `ip3d` and `sip3d` have been removed as they are not computed.

#### PR validation:

Local tests with wfs 1325.81, 136.8523, and 136.88811.

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

These changes are already included in the back port PR #33992 to 10_6_X.

### CODE REVIEW

**Comment by bainbrid** [comment]
@mariadalfonso @gouskos @crovelli @cavalari @jainshilpi @lfinco

**Comment by gouskos** [comment]
please test

**Comment by mariadalfonso** [comment]
This PR fix the empty variables as reported in  
https://github.com/cms-sw/cmssw/pull/33817#issuecomment-855904331

The miniPFRelIso_chg and all are now filled 
Screen Shot 2021-06-08 at 09 31 12

**Comment by mariadalfonso** [comment]
The DQM plots are now saved for all possible nanos, altought empty for when the lowPtElectron doesn't run.
in principle can be customized in PhysicsTools/NanoAOD/python/nanoDQM_cff.py
with the same set of modifiers used in
https://github.com/cms-sw/cmssw/pull/33817/files#diff-31eff23ea52a2b619e0192f0f8d2e89b7e1740522a692066a66284fc43d547f5R195-R203

**Comment by mariadalfonso** [comment]
urgent

(mark what is needed for the nanov9)

File: PhysicsTools/NanoAOD/python/lowPtElectrons_cff.py [hunk 1]
```diff
@@ -76,9 +88,9 @@
         unbiased = Var("electronID('unbiased')",float,doc="ElectronSeed, pT- and dxy- agnostic BDT (raw) score"),
         ptbiased = Var("electronID('ptbiased')",float,doc="ElectronSeed, pT- and dxy- dependent BDT (raw) score"),
         # Isolation
-        miniPFRelIso_chg = Var("userFloat('miniIsoChg')/pt",float,
+        miniPFRelIso_chg = Var("userFloat('miniIsoChg')",float,
                                doc="mini PF relative isolation, charged component"),
-        miniPFRelIso_all = Var("userFloat('miniIsoAll')/pt",float,
```
Comment (mariadalfonso) [inline PhysicsTools/NanoAOD/python/lowPtElectrons_cff.py:81]: Are these relative isolation or not ? 
In this last PR the division for the pt is removed

**Comment by mariadalfonso** [comment]
one more things we can do for master is to clean these warnings
https://github.com/cms-sw/cmssw/pull/33817/files#r645094126

File: PhysicsTools/NanoAOD/python/lowPtElectrons_cff.py (hunk 1 above)
Comment (bainbrid) [reply PhysicsTools/NanoAOD/python/lowPtElectrons_cff.py:81]: Yes, they are rel. iso. even if the name suggests not. So the /pt is no longer needed. This is now done [here](https://github.com/cms-sw/cmssw/pull/34005/files#diff-31eff23ea52a2b619e0192f0f8d2e89b7e1740522a692066a66284fc43d547f5R44).

**Comment by bainbrid** [comment]
> one more things we can do for master is to clean these warnings
> https://github.com/cms-sw/cmssw/pull/33817/files#r645094126

I should fix this in this PR?

**Comment by mariadalfonso** [comment]
> > one more things we can do for master is to clean these warnings
> > https://github.com/cms-sw/cmssw/pull/33817/files#r645094126
> 
> I should fix this in this PR?

yes, fine for me

**Comment by bainbrid** [comment]
> > > one more things we can do for master is to clean these warnings
> > > https://github.com/cms-sw/cmssw/pull/33817/files#r645094126
> > 
> > 
> > I should fix this in this PR?
> 
> yes, fine for me

after discussing privately, we've agreed to factorise this non-essential fix into a separate PR.

And keep this PR is nano-specific.

**Comment by mariadalfonso** [comment]
> The DQM plots are now saved for all possible nanos, altought empty for when the lowPtElectron doesn't run.
> in principle can be customized in PhysicsTools/NanoAOD/python/nanoDQM_cff.py
> with the same set of modifiers used in
> https://github.com/cms-sw/cmssw/pull/33817/files#diff-31eff23ea52a2b619e0192f0f8d2e89b7e1740522a692066a66284fc43d547f5R195-R203

we agreed to defer this update to another PR in master

**Comment by mariadalfonso** [comment]
+xpog

* this PR fix two variables spotted to be broken
* the feature is already in 10_6 for nanov9

**Comment by qliphy** [comment]
+1

**Pull request merged**